js/iframe-coordinator/libs/client.ts

```diff
@@ -83,8 +84,13 @@ function start() {
       }
     });
 
-    onLinkClick((location: Location) => {
-      sendMessage("navRequest", location);
+    window.addEventListener("click", event => {
+      let target = event.target as HTMLElement;
+      if (target.tagName.toLowerCase() === "a" && event.button == 0) {
+        event.preventDefault();
+        let a = event.target as HTMLAnchorElement;
+        sendMessage("navRequest", a.href);
```

Comment:
We can just send along the URL now when intercepting links and parse it in Elm instead of generating a big object in typescript.